packages/shared/src/components/modals/DiscardActionModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement, MouseEvent } from 'react';
```

Comment:
It would probably be a better idea to create a generic modal for confirmation-related ones.